### PR TITLE
erased previous webpack config file

### PR DESCRIPTION
**Pull request by Ledoux**
given https://github.com/coopcycle/coopcycle-js/issues/13

### CODE REVIEW

**Comment by Ledoux** [comment]
@Atala 
ok seems good like this,
can you test it please before merging ? tx

File: src/index.ejs [hunk 1]
```diff
@@ -1,61 +1,19 @@
...
-            color: #ECF0F1;
-          }
-          .close {
-            color: #fff;
-            text-shadow: none
-          }
-        </style>
+      <title>Coopcycle Client Lib Demo</title>
+      <meta charset="utf-8">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```
Comment (Atala) [inline src/index.ejs]: Maybe we should include bootstrap in the SCSS too? This way we can strip unused properties (?)

File: webpack/dev.config.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const webpack = require('webpack')
+
+const config = require('./config')
+const { contentBase,
+  host,
+  url
+} = require('./server.config.js')
+
+const secretDir = path.join(__dirname, '../secret.sh')
+if (!fs.existsSync(secretDir)) {
```
Comment (Atala) [inline webpack/dev.config.js]: Can we allow settings via env vars too?

Sthg like : `!fs.existsSync(secretDir) || (!env. STRIPE_PUBLISHABLE_KEY && !env. GOOGLE_MAPS_API_KEY)

File: webpack/webpackDevServer.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  url
+} = require('./server.config.js')
+
+new WebpackDevServer(
+  webpack(devConfig),
+  {
+    compress: true,
+    contentBase,
+    headers: {
+      // it is important to not set a wildcard for security reason
```
Comment (Atala) [inline webpack/webpackDevServer.js:17]: 👍

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,7 @@
+.DS_Store
 /node_modules/
 /bin/
 /example/*
 !example/index.html
-.idea
\ No newline at end of file
+.idea
+/secret.sh
```
Comment (Atala) [inline .gitignore:7]: Can you add explanation about the `secret.sh` file in the README?

**Review by Atala — CHANGES_REQUESTED** [review]
Thanks for this PR : one step towards production!

File: .gitignore (hunk 4 above)
Comment (Ledoux) [reply .gitignore:7]: 👍

File: webpack/dev.config.js (hunk 2 above)
Comment (Ledoux) [reply webpack/dev.config.js]: 👍

File: src/index.ejs (hunk 1 above)
Comment (Ledoux) [reply src/index.ejs]: 👍

**Comment by Ledoux** [comment]
@Atala 
ok I made the review given your comments,
tell me when we can :shipit: 
tx

**Pull request merged**